bin/get-docs.sh

```diff
@@ -22,5 +22,11 @@ cp -R tmp/docs/* "$docs"
 cd "$pwd"
 cp -R src/* docs/
 
-printf -- '---\ntitle: Merge Confidence\ndescription: Node versions support in Renovate\n---\n\n' > docs/merge-confidence.md
+{
+  printf -- '---\n'
+  printf -- 'title: Merge Confidence\ndescription: Node versions support in Renovate\n'
```

Comment:
The description doesn't match the page content.

It should probably say something like "Learn about Renovate's Merge Confidence feature" or something similar?

Reply:
```suggestion
  printf -- 'title: Merge Confidence\ndescription: Learn about Renovate's Merge Confidence feature\n'
```

@rarkins WDYT?